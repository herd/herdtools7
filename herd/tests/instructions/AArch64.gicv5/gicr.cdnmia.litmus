AArch64 gicr.cdnmia
{
 [INTID(A)]=(pending:1,enabled:1,affinity:P0,active:0,priority:0);
 [INTID(B)]=(pending:1,enabled:1,affinity:P0,active:0,priority:1);
 [INTID(C)]=(pending:1,enabled:1,affinity:P0,active:0,priority:2);
 [INTID(D)]=(pending:0,enabled:1,affinity:P0,active:0,priority:1);
}
 P0              ;
 GICR X1, CDNMIA ;
exists(0:X1=gicval_t:(valid:1,intid:B) \/ 0:X1=gicval_t:(valid:1,intid:C) \/ 0:X1=gicval_t:(valid:1,intid:D))
