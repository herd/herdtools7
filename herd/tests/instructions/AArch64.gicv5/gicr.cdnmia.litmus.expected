Test gicr.cdnmia Allowed
States 2
0:X1=(valid:0);
0:X1=(intid:A, valid:1);
No
Witnesses
Positive: 0 Negative: 2
Condition exists (0:X1=(intid:B, valid:1) \/ 0:X1=(intid:C, valid:1) \/ 0:X1=(intid:D, valid:1))
Observation gicr.cdnmia Never 0 2
Hash=dfb0f6d4ecae40d095116798ef558f02

